kubernetes/services/nginx-storage.yml.j2

```diff
@@ -30,11 +30,17 @@ spec:
               name:       concent-storage
             - mountPath:  /etc/nginx/conf.d/
               name:       nginx-storage-config
+            - mountPath:  /usr/local/openresty/nginx/conf/snippets/
+              name:       nginx-json-errors
+              readOnly:   true
```

Comment:
Add a comment explaining why we're putting this config into `/usr/` and not into `/etc/nginx`.